differentiable_physics/requirements.txt

```
@@ -0,0 +1,3 @@
+torch
```

Due to usage of `torch.accelerator`:
```suggestion
torch>=2.6
```